framework/Models/EnsembleModel.py

```diff
@@ -331,11 +342,7 @@ def initialize(self,runInfo,inputs,initDict=None):
         self.raiseAnError(IOError, "The 'initialStartModels' xml node is not needed for non-Picard calculations, since the running sequence can be automatically determined by the code! Please delete this node to avoid a mistake.")
       self.raiseAMessage("EnsembleModel connections determined a linear system. Picard's iterations not activated!")
 
-    self.allOutputs = []
     for modelIn in self.modelsDictionary.keys():
-      for modelInOut in self.modelsDictionary[modelIn]['Output']:
-        if modelInOut not in self.allOutputs:
-          self.allOutputs.append(modelInOut)
```

Comment:
I do not remember... Don't I use this?

Reply:
It's now set above, during the check for the outputs and output dims, so we don't have to iterate over it twice.

Reply:
ok